scripts/benchmark/contract_data/erc20.sol

```diff
@@ -0,0 +1,274 @@
+pragma solidity ^0.4.23;
```

Comment:
@carver I actually tried to import the openzeppelin repo as submodule and then directly use their example file but `solc ---allow-paths` was driving me bat shit crazy. Need to revisit. For now, this is basically a manual copy and paste act to resolve all imports and concat things into one file